src/components/Dialog/Dialog.scss

```diff
@@ -30,6 +32,20 @@
   left: 390px;
 }
 
+.tm-modal {
```

Comment:
please use more explicit styler definitions instead of `!important`

Reply:
Fixed witha  container